ui/src/pageLayout/containers/Nav.tsx

```diff
@@ -87,36 +91,38 @@ class SideNav extends PureComponent<Props, State> {
           active={getNavItemActivation(['tasks'], location.pathname)}
         />
         <NavMenu.Item
-          title="Settings"
+          title={`${_.startCase(orgName)} Settings`}
```

Comment:
I think `_.startCase` may sometimes alter the org name in undesirable ways

Reply:
I think it just ensures that the first letter of each word is capitalized

Reply:
removing for this pr